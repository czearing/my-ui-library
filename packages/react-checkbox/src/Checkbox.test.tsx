import * as React from 'react';
import { Checkbox } from './Checkbox';
import { render } from '@testing-library/react';

describe('Button', () => {
  it('renders a basic Button', () => {
    const renderedComponent = render(<Checkbox />);

    expect(renderedComponent.baseElement).toMatchSnapshot();
  });
});
