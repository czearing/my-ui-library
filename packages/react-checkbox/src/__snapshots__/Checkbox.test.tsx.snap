// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders a basic Button 1`] = `
<body>
  <div>
    <input
      type="checkbox"
    />
  </div>
</body>
`;
